sdk/rust/src/error.rs

```diff
@@ -59,8 +59,30 @@ pub enum Error {
 
 	#[error("invalid format for QueryTarget")]
 	InvalidQueryTargetFormat,
+
+	#[error(transparent)]
+	Unspecified { source: DynError },
+}
```

Comment:
Should `UnspecifiedQueryState` be removed from the enum in favor of this? Of course that would be done in a future PR

Reply:
It could. I haven’t gone through and removed all of them, but that’s certainly the goal